### DISCUSSION TITLE
Use inference results as labels and train other models

### DISCUSSION THREAD

**Discussion by Mhhhaster** [open-mmlab/mmocr#1880] [Q&A] [answered]
I use `inferencer = MMOCRInferencer(det='dbnetpp', rec='svtr-small')` under icdar13 dataset and get inference results.
Now I want to use them as labels and train other models like ABCNet.
Is it possible to train and eval my model by calling the module.

Specifically, I now have an array representing images and inference results (including rec_texts, rec_scores, det_polygons and det_scores)

**Comment by gaotongxiao** [comment, marked as answer]
Currently we don't support training a new model directly from the Inferencer in an online manner, but it's possible to dump the inference results into [MMOCR's format](https://mmocr.readthedocs.io/en/dev-1.x/basic_concepts/datasets.html#annotation-format) and use this dataset for training.

**Reply by Mhhhaster** [reply]
thanks, I will try to wirte a  dataloader
